projects/forge-angular/src/lib/autocomplete/autocomplete-proxy.module.ts

```diff
@@ -0,0 +1,10 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { AutocompleteComponent } from './autocomplete.component';
+
+@NgModule({
+	imports: [CommonModule],
+	declarations: [AutocompleteComponent],
+	exports: [AutocompleteComponent]
+})
+export class ForgeAutocompleteProxyModule {}
```

Comment:
Here's an example of where a module already existed (for the value accessor directive) so I created a separate module, rather than adding the component to that one so for v2 it's entirely opt-in.  For v3, we should remove this module, and declare/export from the `ForgeAutocompleteModule` so it's not necessary to import both.